src/Scripting/Compiler/BytecodeEmitter.cpp

```
@@ -0,0 +1,911 @@
+
+
+			// Node Links
+			for (const auto &link : nodeLinks) {
+				// Cache outgoing links
+				if (!m_outLinkCache.contains(link.outNodeID))
+					for (const auto& otherLink : nodeLinks)
+						if (link.outNodeID == otherLink.outNodeID)
+							m_outLinkCache[link.outNodeID].emplace_back(otherLink);
+
+				// Cache incoming links
+				if (!m_inLinkCache.contains(link.inNodeID))
```

Cppcheck (2.21.0)

[error] Array 'tmp[1]' accessed at index 1, which is out of bounds.

(arrayIndexOutOfBounds)